code/datums/dna.dm

```diff
@@ -596,3 +556,59 @@
 	return value
 
 /////////////////////////// DNA HELPER-PROCS
+
+/mob/living/carbon/human/proc/something_horrible(ignore_stability)
+	if(!has_dna()) //shouldn't ever happen anyway so it's just in really weird cases
+		return
+	if(!ignore_stability && (dna.stability > 0))
+		return
+	var/instability = -dna.stability
+	dna.remove_all_mutations()
+	dna.stability = 100
+	if(prob(max(70-instability,0)))
+		switch(rand(0,6)) //not complete and utter death
+			if(0)
+				monkeyize()
+			if(1)
+				gain_trauma(/datum/brain_trauma/severe/paralysis/paraplegic)
+				new/obj/vehicle/ridden/wheelchair(get_turf(src)) //don't buckle, because I can't imagine to plethora of things to go through that could otherwise break

```

Comment:
I think this is missing a space. Like `new /obj`

Reply:
It doesn't matter wheter there's a space or not

Reply:
what the fuck dm

give it a space